test/metabase/api/card_test.clj

```diff
@@ -139,7 +139,7 @@
                         {:database (u/the-id db)
                          :type     :native
                          :native   {:query         "SELECT COUNT(*) FROM VENUES WHERE CATEGORY_ID = {{category}};"
-                                    :template-tags {:category {:id           "a9001580-3bcc-b827-ce26-1dbc82429163"
```

Comment:
These `:id` changes aren't meaningful but they make test output a little easier to look at.